package.json

```diff
@@ -56,6 +56,7 @@
     "test:storefront:lib": "ng test storefrontlib --sourceMap --code-coverage",
     "dev:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 ng run storefrontapp:serve-ssr",
     "serve:ssr": "node dist/storefrontapp-server/main.js",
+    "serve:ssr:ci": "NODE_TLS_REJECT_UNAUTHORIZED=0 SSR_TIMEOUT=0 node dist/storefrontapp-server/main.js",
```

Comment:
will it work cross-env? I mean Windows/Linux?

Reply:
It should. Can you ask someone to test it on Windows?